ansible/initialize-master.yml

```
@@ -56,10 +56,32 @@
       until: result is not failed
       when: initialized is changed
 
+      # Azure doesn't support IP-in-IP so run calico in VXLAN mode
+    - name: Set calico to use VXLAN instead of IP-in-IP
+      command: kubectl --kubeconfig /etc/kubernetes/admin.conf patch daemonsets. -n kube-system calico-node --patch "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"calico-node\",\"env\":[{\"name\":\"CALICO_IPV4POOL_IPIP\",\"value\":\"Never\"},{\"name\":\"CALICO_IPV4POOL_VXLAN\",\"value\":\"Always\"}]}]}}}}"
+      when: cloud_provider is defined and cloud_provider == "azure"
```

Perhaps add this to be more in line with the other calico tasks.
```suggestion
      when: initialized is changed and cloud_provider is defined and cloud_provider == "azure"
```